spec/models/post_action_spec.rb

```diff
@@ -501,6 +501,37 @@ def value_for(user_id, dt)
       expect(post.hidden_reason_id).to eq(Post.hidden_reasons[:flagged_by_tl3_user])
     end
 
+    it "hide non-tl4 posts that are flagged by a tl4 user" do
+      Discourse.stubs(:site_contact_user).returns(admin)
```

Comment:
Note... I much prefer to do this via site settings, can you do a follow up commit